modules/lead/product-jenkins/pipelines.tpl

```diff
@@ -0,0 +1,8 @@
+{
+%{ for name,pipeline in pipelines ~}
+  ${name}: {
+    type: ${pipeline.type},
+    repo: ${pipeline.repo},
+    org: ${pipeline.org}
+  },
```

Comment:
i worry you will end up with invalid JSON due to the extra `,` here

Reply:
for now I could do something like a substr to remove the extra `,`

Reply:
did you test it?  if it works, then i'm good going forward with it as is

Reply:
yup,  confirmed it works locally and just pushed up the change